<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-GRUOEUJ" failures="1" tests="7" name="Personal Loan" time="0.187" errors="0" timestamp="2022-10-31T18:14:50 IST">
  <testcase classname="Practice.day2" name="dataclass" time="0.012"/>
  <testcase classname="Practice.day2" name="dataclass" time="0.008"/>
  <testcase classname="Practice.day2" name="dataclass" time="0.002"/>
  <testcase classname="Practice.day2" name="day" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Practice.day2.day(day2.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- day -->
  <testcase classname="Practice.day4" name="Mobilelogin" time="0.003"/>
  <testcase classname="Practice.day4" name="apilogin" time="0.003"/>
  <testcase classname="Practice.day4" name="webloginHomeLoan" time="0.002"/>
  <testcase classname="Practice.day1" name="test" time="0.0">
    <ignored/>
  </testcase> <!-- test -->
  <testcase classname="Practice.day1" name="Prarameters" time="0.0">
    <ignored/>
  </testcase> <!-- Prarameters -->
</testsuite> <!-- Personal Loan -->
